src/aoptk/relationships/zero_shot_classification.py

```diff
@@ -0,0 +1,77 @@
+from __future__ import annotations
+from itertools import product
+from typing import TYPE_CHECKING
+from transformers import pipeline
+from aoptk.chemical import Chemical
+from aoptk.effect import Effect
+from aoptk.relationships.find_relationship import FindRelationships
+from aoptk.relationships.relationship import Relationship
+
+if TYPE_CHECKING:
+    from aoptk.chemical import Chemical
+    from aoptk.effect import Effect
+
+
+class ZeroShotClassification(FindRelationships):
+    """Zero-shot classification for finding relationships between chemicals and effects in text."""
+
+    task = "zero-shot-classification"
+    threshold = 0.6
+    margin = 0.15
+
+    def __init__(
+        self,
+        text: str,
+        chemicals: list[Chemical],
+        effects: list[Effect],
```

Comment:
These three parameters should move to the `find_relationships` function, as defined in the interface :)